### PR TITLE
More cleanup needed without core in maven repo

### PR DESCRIPTION
**Pull request by mfrazier63**
## Description
One last bit of cleanup found when maven repo is purged of horizon core jars.
## Jira link(s)
- https://issues.opennms.org/browse/HS-xx

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: minion/minion-3rdParty/shaded-grpc/pom.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                            io.grpc.stub,
+                            io.grpc.protobuf,
+                            io.grpc.internal,
+                            io.grpc.netty.shaded.io.netty.util.collection,
+                            io.grpc.netty.shaded.io.grpc.netty,
+                            io.grpc.netty.shaded.io.netty.handler.ssl,
+                            io.grpc.netty.shaded.io.netty.util.collection
+                        </Export-Package>
+                    </instructions>
+                    <unpackBundle>true</unpackBundle>
```
Comment (splatch) [inline minion/minion-3rdParty/shaded-grpc/pom.xml:87]: I don't know that instruction, do you know what it causes?

File: minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java [hunk 2]
```diff
@@ -30,39 +30,39 @@
...
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlAttribute;
-import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
+//import javax.xml.bind.annotation.XmlAccessType;
+//import javax.xml.bind.annotation.XmlAccessorType;
+//import javax.xml.bind.annotation.XmlAttribute;
+//import javax.xml.bind.annotation.XmlElement;
+//import javax.xml.bind.annotation.XmlRootElement;
```
Comment (splatch) [inline minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java:37]: Please remove these lines permanently.

File: minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java [hunk 3]
```diff
@@ -30,39 +30,39 @@
...
 import org.opennms.horizon.shared.ipc.rpc.api.RpcRequest;
 
-@XmlRootElement(name="echo-request")
-@XmlAccessorType(XmlAccessType.NONE)
+//@XmlRootElement(name="echo-request")
+//@XmlAccessorType(XmlAccessType.NONE)
 public class EchoRequest implements RpcRequest {
 
-    @XmlAttribute(name="id")
+//    @XmlAttribute(name="id")
```
Comment (splatch) [inline minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java:47]: Please remove jaxb statements, just like we did with snmp stuff.

**Review by splatch — APPROVED** [review]
Just evict jaxb stuff and we're good to go.

File: minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java (hunk 2 above)
Comment (mfrazier63) [reply minion/minion-ipc/echo/src/main/java/org/opennms/horizon/minion/ipc/echo/EchoRequest.java:37]: - [ ] Removed commented out annotations

File: minion/minion-3rdParty/shaded-grpc/pom.xml (hunk 1 above)
Comment (mfrazier63) [reply minion/minion-3rdParty/shaded-grpc/pom.xml:87]: It changes the import and export statements in the bundle manifest.

**Pull request merged**